Lib/test/test_fcntl.py

```diff
@@ -164,6 +164,7 @@ def test_lockf_exclusive(self):
         fcntl.lockf(self.f, fcntl.LOCK_UN)
         self.assertEqual(p.exitcode, 0)
 
+    # Passes on OS400...
     @unittest.skipIf(platform.system() == "AIX", "AIX returns PermissionError")
```

Comment:
Could remove this skip too and needs no comment.

Reply:
[Upstream](https://github.com/kadler/cpython/blob/master/Lib/test/test_fcntl.py#L158-L178) has this and https://github.com/kadler/cpython/pull/47/files#r644078470  skipped on AIX. Should we remove this skip though? I'm thinking we remove the comments and remove the skips for IBM i.

Reply:
If we fix the internal flock and lockf implementations, then the original bug is fixed and it no longer needs to be skipped.

Reply:
Removed checks in https://github.com/kadler/cpython/pull/47/commits/d788d966126a130bb7ba3218517a7615a8974c6a. No new tests were failing after running a build